osutil/user_test.go

```diff
@@ -279,6 +280,82 @@ func (s *createUserSuite) TestIsValidUsername(c *check.C) {
 	}
 }
 
+type delUserSuite struct {
+	mockUserDel *testutil.MockCmd
+	opts        *osutil.DelUserOptions
+	sudoersd    string
+	restore     func()
+}
+
+var _ = check.Suite(&delUserSuite{opts: nil})
+var _ = check.Suite(&delUserSuite{opts: &osutil.DelUserOptions{ExtraUsers: true}})
+
+func (s *delUserSuite) SetUpTest(c *check.C) {
+	s.mockUserDel = testutil.MockCommand(c, "userdel", "")
+	s.sudoersd = c.MkDir()
+	s.restore = osutil.MockSudoersDotD(s.sudoersd)
```

Comment:
(nitpick^2) I think I would slightly prefer testutil.BaseTest and AddCleanup here but not worth changing, it's such a detail :)